tooling/build-types-and-manifests.ts

```diff
@@ -144,7 +144,20 @@ const makeLocalFesBuild = (sourceBuildType: string) => {
   );
 };
 
+const makeContentScriptTestsBuild = (sourceBuildType: string) => {
+  const testCode = readFileSync('./test/source/tests/content-script-test.js').toString();
+  const testBuildType = sourceBuildType.endsWith('-mock')
+    ? sourceBuildType.slice(0, -5) + '-content-script-tests-mock'
+    : sourceBuildType + '-content-script-tests';
+  exec(`cp -r ${buildDir(sourceBuildType)} ${buildDir(testBuildType)}`);
+  edit(`${buildDir(testBuildType)}/js/content_scripts/webmail_bundle.js`, code =>
```

Comment:
This basically wipes out the webMail code, only libraries are left and `[test/source/tests/content-script-test.js](https://github.com/FlowCrypt/flowcrypt-browser/pull/5013/files#diff-df3637596ba657872526c5b17037eaae42f0734b1ae951bdcf1ba7efbdb8f2fd)
immediately calls `OpenPGP.js` and `web-stream-tools` directly